apps/web/src/routes/admin/settings.tsx

```
@@ -0,0 +1,327 @@
+            >
+              <div className="flex items-center justify-between">
+                <div className="flex items-center gap-3">
+                  <button
+                    onClick={() => handleToggleWebhook(webhook)}
+                    disabled={togglingWebhookId === webhook.id}
+                    className="text-gray-400 hover:text-gray-600 transition-colors"
+                  >
+                    {webhook.enabled ? (
+                      <ToggleRight className="w-6 h-6 text-green-500" />
+                    ) : (
+                      <ToggleLeft className="w-6 h-6" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)